plugins/memory/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "verdaccio-memory",
+  "version": "2.0.0",
+  "description": "Storage implementation in memory",
+  "keywords": [
+    "verdaccio",
+    "plugin",
+    "storage",
+    "memory"
+  ],
+  "author": "Juan Picado <juanpicado19@gmail.com>",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib/"
+  ],
+  "engines": {
+    "node": ">=8"
+  },
+  "dependencies": {
+    "@verdaccio/streams": "2.0.0",
+    "http-errors": "1.7.2",
+    "memory-fs": "0.4.1"
+  },
+  "devDependencies": {
+    "@verdaccio/babel-preset": "^2.0.0",
+    "@verdaccio/eslint-config": "^2.0.0",
+    "@verdaccio/types": "^5.3.0"
```

Comment:
I don't update to latest version (7.0.0) because it requires to define some methods about tokens and I don't have too much knowledge about

Reply:
No worries, I'll take of it later on.